### DISCUSSION TITLE
[indexer] Get ready to nearcore 1.23.1

### DISCUSSION THREAD

**Discussion by khorolets** [near/nearcore#5912] [Node Public Interfaces]
`nearcore` is planned to be updated to the version of 1.23.1:

* `testnet` on Jan 17, 2022
* `mainnet` on Feb 7, 2022

## Release Candidate 1
Protocol upgrade is expected, it means that if you have your indexer outdated it'll be stuck once a protocol will be upgraded. In order to prevent it, please update your indexers before Dec 23 (the earlier the better)

### Quick info
---
**Commit hash** [`4807978dd6a952a25b2f304fb15433ea9480b39d `](https://github.com/near/nearcore/commit/4807978dd6a952a25b2f304fb15433ea9480b39d) 1.23.1-rc.1
**Expected issues**: None
**Reference**: have a look at how [we made it on NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer/pull/216)

## Release Candidate 2

A new RC has been created, the commit that might introduce the huge RAM usage was reverted.

### Quick info
---
**Commit hash** [`8d00e3774e753b198b7c71fbaf6476ba8f6a4f7a `](https://github.com/near/nearcore/commit/8d00e3774e753b198b7c71fbaf6476ba8f6a4f7a) 1.23.1-rc.2
**Expected issues**: None
**Reference**: have a look at how [we made it on NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer/pull/221/files)

Feel free to leave your questions in the comments below.

**Comment by DDeAlmeida** [comment]
Thank you. Got an error on Indexer for guildnet.
Going to update it.

**Reply by khorolets** [reply]
Would you share what error you did you have and how you solved it?

**Reply by DDeAlmeida** [reply]
Sorry i didn't take a snapshot. But got something like inconsistent version etc
So i update the indexer and now i have:
![image](https://user-images.githubusercontent.com/52310650/147003373-f2346c29-0c96-4963-ab04-6849ff9f67e7.png)
Now only  " INFO stats: State `EQzx2yL34m6m9uKAQoHHSC2U4rQFovinkUrj9XLMfHc2`[0: parts][1: parts][2: parts][3: parts][4: parts][5: parts][6: parts][7: parts] " showed :s

Need to remove data and sync again ?

**Reply by DDeAlmeida** [reply]
So i remove all data and resync. Works

**Comment by khorolets** [comment]
Please note that the original post was updated with info about RC2
